jdl/converters/jdl-to-json/jdl-to-json-option-converter.ts

```diff
@@ -30,22 +37,22 @@ const serviceClassOptionValue = BinaryOptions.Values.service.SERVICE_CLASS;
 
 const USER = 'user';
 
-let convertedOptionContent;
+let convertedOptionContent: Map<string, any>;
+
+export default { convert };
 
-module.exports = {
-  convert,
-};
+type JDLOptionHolder = JDLObject | JDLApplication;
 
 /**
  * Converts JDL options to JSON content to be set in JSON entities.
  * This function takes a single parameter: a JDL option holder. This holder is a class or an object responding to these
  * calls:
  *   - forEachOption, to loop over JDL options
  *   - getEntityNames, to get the declared entity names
- * @param {JDLObject|JDLApplication} jdlOptionHolder - a JDL object (a JDLObject or a JDLApplication) containing the options.
+ * @param jdlOptionHolder - a JDL object (a JDLObject or a JDLApplication) containing the options.
```

Comment:
I didn't notice before but this change makes me realize that we can remove the comments altogether, as the IDEs can make use of TS typing system to assist the developers in knowing the signatures, etc.

We can have another issue to remove comments from this part of the code :)

Reply:
What part of the comment do you mean exactly? 

We can remove the types in the JSDocs after the types are annotated using TypeScript. That is what I've done in this line and in some other places.

Reply:
All the comments :) they're not useful

Reply:
The comments do provide important information in some cases and I think having good comments makes the code easier to work with. But obviously not every comment is useful.